salt-minion/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:trusty
+MAINTAINER Sascha F. Andreichenko <andreichenko@web.de>
+RUN apt-get update && apt-get install -y salt-minion curl
+COPY setup.sh /opt/setup.sh
```

Comment:
is it necessary if script will be copied to /opt folder? @freideveloper

Reply:
Item #2 present an issue, because this means you have 250 seconds to create all of your minion containers, and then link them to the master, which, if you feel like writing a bunch of bash to do this, then sure, go for it, but the goal here is to orchestrate, so here's where Docker Compose comes in to handle, not only the dependency I'm describing, but also the linking required to resolve the service (the fleet of containers of the same type--similar to the container definitions within a Kubernetes Pod, if you've never worked with Docker directly before, or the components in a LAMP stack if you've never dealt with a container runtime at all but may be familiar with a stack as being different components as a single logical application).